tests/GridFS/BucketFunctionalTest.php

```diff
@@ -160,6 +160,41 @@ public function testDeleteStillRemovesChunksIfFileDoesNotExist($input, $expected
         $this->assertCollectionCount($this->chunksCollection, 0);
     }
 
+    public function testDeleteByName(): void
```

Comment:
Should this Pr block on spec changes for [DRIVERS-2807](https://jira.mongodb.org/browse/DRIVERS-2807) and [DRIVERS-2808](https://jira.mongodb.org/browse/DRIVERS-2808)? Also, will the PR(s) for those tickets include spec tests to be synced?

Reply:
Spec tests added in https://github.com/mongodb/specifications/pull/1702